packages/app-earn-ui/src/components/molecules/DateRangePicker/DateRangePicker.module.css

```
@@ -0,0 +1,88 @@
+  color: var(--color-background-primary-hover);
+}
+
+.datePicker :global(.rdp-selected) {
+  color: var(--color-background-primary-hover);
+  background-color: unset;
+}
+
+.datePicker :global(.rdp-range_start) {
+  color: var(--color-background-primary-hover);
+  background-color: unset;
+  background: unset;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)